wallet-connect/package.json

```diff
@@ -0,0 +1,56 @@
+{
+	"name": "sigmadao-ui",
+	"description": "Sigma DAO UI",
+	"homepage": "https://algobuilder.dev",
+	"license": "mpl-2.0",
+	"engines": {
+		"node": ">=14.11.0",
+		"yarn": ">=3.1.0"
+	},
+	"scripts": {
+		"dev": "vue-cli-service serve",
+		"build": "vue-cli-service build",
+		"lint": "yarn lint:js && yarn lint:prettier",
+		"start": "node .output/server/index.mjs",
+		"lint:js": "eslint --ext \".js,.ts,.vue\" --ignore-path .gitignore .",
+		"lint:prettier": "prettier --check .",
+		"lint:fix": "prettier --write --list-different . && yarn lint:js --fix",
+		"upgrade:dependency": "npx yarn-upgrade-all",
```

Comment:
let's remove this. We have monorepo and this should be done by `yarn up` in the root.

Reply:
Done